jobs/build/update-base-images/build.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
I belive comman.shell() already set this up, but from the jenkins log I didnt see any setup... so just add it here to double-check